base/db/R/insert.format.vars.R

```diff
@@ -126,13 +126,13 @@ insert.format.vars <- function(con, format_name, mimetype_id, notes = NULL, head
       u1 <- formats_variables[1,"unit"]
       u2 <- dplyr::tbl(con, "variables") %>% dplyr::select(.data$id, units) %>% dplyr::filter(.data$id %in% !!formats_variables[[1, "variable_id"]]) %>% dplyr::pull(.data$units)
 
-      if(!udunits2::ud.is.parseable(u1)){
+      if(!units::install_unit(u1)){
```

Comment:
`install_unit` is for defining new units that aren't defined by the udunits library. I think the better replacement for `ud.is.parseable` would be to try `as_units(u1)` and treat failure as meaning it's unparseable.

Reply:
It's also possible we could skip this step and let the failure happen inside `ud_are_convertible` immediately below? Need to think more abou this